airbyte-cdk/java/airbyte-cdk/airbyte-commons/src/main/java/io/airbyte/commons/concurrency/CompletableFutures.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.commons.concurrency;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+public class CompletableFutures {
+
+  /**
+   * Utility method which blocks until all futures are complete. Returns a list of the results of all
+   * futures.
+   *
+   * @param futures
+   * @return
+   * @param <T>
+   */
+  public static <T> List<T> allOf(final List<CompletableFuture<T>> futures) {
```

Comment:
is this file needed? not in use afaict, and the implementation looks broken

Reply:
Created for something else but removed the impl for now on this branch. I'll remove it later if not needed.